java/ingest/ingest-runner/src/main/java/sleeper/ingest/job/IngestJobQueueConsumerRunner.java

```diff
@@ -82,8 +110,21 @@ public static void main(String[] args) throws InterruptedException, IOException,
         ObjectFactory objectFactory = new ObjectFactory(instanceProperties, s3Client, "/tmp");
 
         String localDir = "/mnt/scratch";
+        TablePropertiesProvider tablePropertiesProvider = new TablePropertiesProvider(s3Client, instanceProperties);
+        StateStoreProvider stateStoreProvider = new StateStoreProvider(dynamoDBClient, instanceProperties, HadoopConfigurationProvider.getConfigurationForECS(instanceProperties));
```

Comment:
It looks like this is the same Hadoop configuration that was used for the StateStoreProvider before, but I wonder if it should be merged with the other one so we don't have two separate configurations.

Reply:
The S3 state store isn't particularly performance sensitive, so we can use whatever configuration we set for the ingest for the state store as well.